Sources/Moya/MoyaProvider+Internal.swift

```diff
@@ -107,6 +97,28 @@ public extension MoyaProvider {
     // swiftlint:enable cyclomatic_complexity
     // swiftlint:enable function_body_length
 
+    private func unstubbedRequest(_ target: Target, request: URLRequest, callbackQueue: DispatchQueue?, progress: Moya.ProgressBlock?, completion: @escaping Moya.Completion) -> Cancellable {
```

Comment:
I wouldn't know what this function does without checking the source code, unfortunately. I'm thinking if we can create a function that would return cancellable, but also it would check the `stubBehavior` and base on that one. Like:
```swift
func abc(...) -> Cancellable {
    switch stubBehavior {
        ...
    }
}
```
so we could remove this logic in `performNetworking` and just assign the `Cancellable`:
```swift
cancellableToken.innerCancellable = abc(...)
```

it's responsibility would be to send a request based on types & return `Cancellable`. What do you think?